jsk_aibo_robot/aibo_selenium_ros/README.md

```diff
@@ -0,0 +1,67 @@
+# aibo_selenium_ros
+
+This package provides a ROS Node which retieve head camera images from web browser and publish it as ROS message.
+
+## How to setup
+
+**Currently this node is only tested with ROS noetic.**
+
+First, Install [Google chrome](https://www.google.com/intl/ja_jp/chrome/) to your PC.
+
+Second, Download [suitable version of chrome webdriver to Google chrome installed](https://chromedriver.chromium.org/downloads) and place it somewhere.
```

Comment:
is there any reason you do not use system `chromium-browser` and`chromium-chromedriver`

Reply:
Changed to chromium and use it.